docs/guides/how_to_guides/creating_and_editing_expectations/how_to_create_an_expectation_suite_with_the_user_configurable_profiler.rst

```diff
@@ -0,0 +1,250 @@
+
+            .. code-block:: python
+
+                profiler = UserConfigurableProfiler(dataset=batch)
+
+
+        6. Use the profiler to build a suite
+
+            Now that we have our profiler set up with our batch, we can use the build_suite method on the profiler. This will print a list of all the expectations created by column, and return the Expectation Suite object.
+
+            .. code-block:: python
+
+                suite = profiler.build_suite()
+
+        7. (Optional) Running validation, saving the suite and building Data Docs
+
+            If you'd like, you can validate the your data with the new suite, save your Expectation Suite and build Data Docs to take a closer look
+
+            .. code-block:: python
+                # We need to re-create our batch to link the batch with our new suite
+                batch = context.get_batch(
+                batch_kwargs=batch_kwargs,
+                expectation_suite_name=suite)
+
+                # Running validation
+                results = context.run_validation_operator("action_list_operator", assets_to_validate=[batch])
+                validation_result_identifier = results.list_validation_result_identifiers()[0]
+
+                # Saving our expectation suite
+                context.save_expectation_suite(suite, expectation_suite_name)
+
+                # Building and opening Data Docs
+                context.build_data_docs()
+                context.open_data_docs(validation_result_identifier)
+
+
+        **Optional Parameters**
+
+        The UserConfigurableProfiler can take a few different parameters to further hone the results. These parameter are:
+
+            - ``excluded_expectations``: Takes a list of expectation names which you want to exclude from the suite
+
+
+            - ``ignored_columns``: Takes a list of columns for which you may not want to build expectations (i.e. if you have metadata which might not be the same between tables
+
+
+            - ``not_null_only``: Takes a boolean. By default, each column is evaluated for nullity. If the column values contain fewer than 50% null values, then the profiler will add ``expect_column_values_to_not_be_null``; if greater than 50% it will add ``expect_column_values_to_be_null``. If ``not_null_only`` is set to True, the profiler will add a not_null expectation irrespective of the percent nullity (and therefore will not add an ``expect_column_values_to_be_null``)
+
+
+            - ``primary_or_compound_key``: Takes a list of one or more columns. This allows you to specify one or more columns as a primary or compound key, and will add ``expect_column_values_to_be_unique`` or ``expect_compound_column_values_to_be_unique``
+
+
+            - ``table_expectations_only``: Takes a boolean. If True, this will only create table-level expectations (i.e. ignoring all columns. Table-level expectations include ``expect_table_row_count_to_equal`` and ``expect_table_columns_to_match_ordered_list``
```

Comment:
```suggestion
            - ``table_expectations_only``: Takes a boolean. If True, this will only create table-level expectations (i.e. ignoring all columns). Table-level expectations include ``expect_table_row_count_to_equal`` and ``expect_table_columns_to_match_ordered_list``
```